### DISCUSSION TITLE
Getting  - Error: Unexpected number of middleware 2 when deploying to Vercel

### DISCUSSION THREAD

**Discussion by gavriguy** [vercel/community#1290] [Help] [answered]
### Summary

When running a  prebuilt deploy I'm getting: `Error: Unexpected number of middleware 2`
Any idea why?

```
Vercel CLI 28.12.7
Inspect: https://vercel.com/.../CuGXsrrAPmx4bev5qGytZtKXyPMN [7s]
Error: Unexpected number of middleware 2
```

**Comment by gavriguy** [comment]
I'm running a remix app, if it matters.

**Comment by mmachatschek** [comment]
also seeing this issue on our remix app after our system switched to the new 28.12.7 version.

before we had this error and hat to lock the cli version: https://github.com/orgs/vercel/discussions/1282#discussioncomment-4718436

**Comment by swarnava** [comment]
We are looking into this currently.

**Comment by swarnava** [comment, marked as answer]
We have pushed a fix for this ~30 mins ago. This should be resolved now.
